fluent-dom/src/registry.js

```diff
@@ -0,0 +1,24 @@
+import { MessageContext } from '../../fluent/src/index';
```

Comment:
Remove this file from this PR, please. It should be part of the planned `fluent-web` package which glues other `fluent-*` packages together for the purpose of using them in HTML documents.